gallery/src/demos/demo-hui-media-player-rows.ts

```diff
@@ -55,26 +61,23 @@ const CONFIGS = [
   },
 ];
 
-class DemoHuiMediaPlayerRows extends PolymerElement {
-  static get template() {
-    return html` <demo-cards id="demos" configs="[[_configs]]"></demo-cards> `;
-  }
+@customElement("demo-hui-media-player-card")
+class DemoHuiMediaPlayerRows extends LitElement {
+  @internalProperty() private _configs = CONFIGS;
+
+  @query("#demos") private _demoRoot!: HTMLElement;
 
-  static get properties() {
-    return {
-      _configs: {
-        type: Object,
-        value: CONFIGS,
-      },
-    };
+  protected render(): TemplateResult {
+    return html`<demo-cards id="demos" .configs=${this._configs}></demo-cards>`;
   }
 
-  public ready() {
-    super.ready();
-    const hass = provideHass(this.$.demos);
+  protected firstUpdated(changedProperties: PropertyValues) {
+    super.firstUpdated(changedProperties);
+    const hass = provideHass(this._demoRoot);
     hass.updateTranslations(null, "en");
+    hass.updateTranslations("lovelace", "en");
     hass.addEntities(createMediaPlayerEntities());
   }
 }
 
-customElements.define("demo-hui-media-player-rows", DemoHuiMediaPlayerRows);
+customElements.define("demo-hui-media-player-card", DemoHuiMediaPlayerRows);
```

Comment:
Name of the file is not different from the element?

Reply:
Copy&paste oversight